plugin/trino-hive/src/test/java/io/trino/plugin/hive/s3/TestTrinoS3FileSystem.java

```diff
@@ -812,27 +808,24 @@ public void testStreamingUpload()
     {
         Configuration config = newEmptyConfiguration();
         config.set(S3_STREAMING_UPLOAD_ENABLED, "true");
-        config.set(S3_STREAMING_UPLOAD_PART_SIZE, "10");
```

Comment:
this seems important for testing

Reply:
I will modify this test, tough it doesn't seem correct to set part size to smaller value then initial buffer size that we set to 64.

Reply:
>  then initial buffer size that we set to 64.

but you set it to 64, right?